CodeEdit/Features/Settings/Pages/TerminalSettings/Models/TerminalSettings.swift

```diff
@@ -5,6 +5,7 @@
 //  Created by Nanashi Li on 2022/04/08.
 //
 
+import AppKit.NSFont
```

Comment:
No need to import the specific NSFont module there's no benefit, remove this diff to keep the imports simpler.